### PR TITLE
Add support for new `build_id` output

### PR DESCRIPTION
**Pull request by eBardX**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,29 +48,34 @@ specified as a _secret_ environment variable.
   If this mode is enabled, additional debug information is printed. (Default:
   `false`.)
 
+## Inputs
```
Comment (VincentJaouen) [inline README.md]: Don't you mean `## Outputs` ?

**Review by VincentJaouen — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (eBardX) [reply README.md]: Good eye! I will fix.

**Review by laurentsigal — COMMENTED** [review]
Amazing pace!!

**Pull request merged**